data/actionsmap/yunohost.yml

```diff
@@ -925,6 +925,57 @@ monitor:
 
 
 #############################
+#         Settings          #
+#############################
+settings:
+    category_help: Manage YunoHost global settings
+    actions:
+
+        ### settings_list()
+        list:
+            action_help: list all entries of the settings
+            api: GET /settings
+
+        ### settings_get()
+        get:
+            action_help: get an entry value in the settings
+            api: GET /settings/<key>
+            arguments:
+                key:
+                    help: Settings key
+
+        ### settings_set()
+        set:
+            action_help: set an entry value in the settings
+            api: POST /settings/<key>
+            arguments:
+                key:
+                    help: Settings key
+                -v:
+                    full: --value
+                    help: new value
+                    extra:
+                        required: True
+
+        ### settings_reset()
+        reset:
+            action_help: reset all settings to their default value
+            api: DELETE /settings
+            arguments:
+                -y:
+                    full: --yes
+                    help: Yes I'm sure I want to do that
+                    action: store_true
+
+        ### settings_default()
+        default:
```

Comment:
Ok.

On the other hand that makes me thing that "cert-install" and related commands should also have a "--all" flag, when commands doesn't have something like that I often end up writing the command without arguments to have its arguments displayed and end up running it for all possible arguments :x